backend/src/main/java/de/jmpsoftware/backend/controller/FixtureController.java

```diff
@@ -29,65 +30,76 @@ public FixtureController(DMXService dmxService) {
 
 
     @GetMapping(path = "/allfixturetemplatelist")
-    public List<String> returnAllFixtureTemplateList() {
-        return dmxService.getAllFixtureTemplateList();
-    }
+    public ResponseEntity<List<String>> returnAllFixtureTemplateList() {return ResponseEntity.ok( dmxService.getAllFixtureTemplateList());}
 
     @ResponseBody
     @PostMapping(path = "/createfixture")
-    public void postFixtureFader(@RequestBody CreateFixtureItem fixtureItem) {
+    public ResponseEntity<String> postFixtureFader(@RequestBody CreateFixtureItem fixtureItem) {
         if (fixtureItem == null)
-            return;
+            throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "Not fixtureItem" );
 
         dmxService.createFixtureFromTemplateList(fixtureItem.getTemplateName(),  fixtureItem.getFixtureName(), fixtureItem.getAddress(), fixtureItem.getUniverse());
+        return ResponseEntity.ok().build();
     }
 
 
   @GetMapping(path = "/allactivefixture")
-    public List<ActiveFixtureItem> returnAllActiveFixture() {
-        return dmxService.getAllActiveFixture();
+    public ResponseEntity<List<ActiveFixtureItem>> returnAllActiveFixture() {
+      return ResponseEntity.ok(dmxService.getAllActiveFixture());
+    }
+
+    @ResponseBody
+    @PutMapping(path = "/deleteactivefixture")
```

Comment:
I would use the delete http method for deleting objects. If you realy need the send data in the body i would at least use a post.